### PR TITLE
increase groovy version

### PR DESCRIPTION
**Pull request by wow-such-code**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,8 +14,8 @@
 	<packaging>jar</packaging>
 
     <properties>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```
Comment (jenniferboedker) [inline pom.xml]: why change it to java 8/why was it java 11 in the first place?

Comment (wow-such-code) [reply pom.xml]: Changed in maintenance, not sure why the actions were running - the dependencies here need java 8, we had this problem multiple times in the past :/

Comment (KochTobi) [reply pom.xml]: what exactly is not working?

Comment (wow-such-code) [reply pom.xml]: nevermind, I confused two issues here

**Review by jenniferboedker — APPROVED** [review]

**Pull request merged**